hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1297,29 +1374,55 @@
       tests: []
     }
 
-    ////////////////////////////////////////////////
-    // Memory & Controllers                       //
-    // ROM, RAM, FLASH, FLASH_CTRL, OTP, OTP_CTRL //
-    ////////////////////////////////////////////////
+    /////////////////////////////////////////////////////
+    // Memory & Controllers                            //
+    // ROM_CTRL, RAM, FLASH, FLASH_CTRL, OTP, OTP_CTRL //
+    /////////////////////////////////////////////////////
 
-    // ROM (pre-verified IP) integration tests:
-    // TODO: Not sure if this will really be a pre-verified IP. If not, then more tests are needed
-    // at the chip level.
+    // ROM_CTRL (pre-verified IP) integration tests:
     {
       name: chip_rom_access
-      desc: '''Verify access to the rom.
+      desc: '''Verify that the CPU can access the rom contents.
 
-            Verify that the CPU can fetch instructions from the ROM. Nothing extra needs to be done
-            here - all SW tests do this anyway.
+            - Verify that the CPU can fetch instructions from the ROM.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_rom_security_features
-      desc: '''Verify ROM security / ECC features if available.
+      name: chip_rom_ctrl_ast_cfg
+      desc: '''Verify that the connectivity of AST cfg signals to the ROM ctrl.
 
-            Details TBD.
+            - In open source, this is verified by a simple connectivity assertion check.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_rom_ctrl_integrity_check
+      desc: '''Verify that the ROM ctrl performs the integrity check of the ROM on power up.
+
+            - In non-PROD LC state, the computed digest does not have to match the top 8 words in
+              the ROM. Verify that we can successfully power up the chip in this case.
+            - In PROD LC state, verify that the pwrmgr does not fully power up if the computed
+              digest does not match the top 8 words of the ROM.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_rom_ctrl_reset_glitch
+      desc: '''Verify that a glitch on the ROM ctrl's reset input is triggered as a fatal alert.
+
+            - In normal boot up from POR, the ROM contents are checked for validity using KMAC.
+            - After boot up, read the reset cause register in rstmgr to confirm POR reset phase.
+            - SW goes into idle state (nothing to do).
+            - The testbench randomly glitches the ROM ctrl's reset input, which causes ROM ctrl to
+              re-check the contents for validity.
+            - Verify that the KMAC detects the repeat presentation of data from ROM ctrl as a fatal
+              alert and the chip is rebooted via the alert escapation path.
```

Comment:
@eunchan, @udinator 
Need to confirm if KMAC will be updated to detect this behavior.

Reply:
to the best of my knowledge currently the design doesn't support this, @eunchan can elaborate more